recon_test_pack/old_scatter_cylinder.hs

```diff
@@ -0,0 +1,50 @@
+!INTERFILE  :=
+name of data file := old_scatter_cylinder.s
+originating system := unknown
+!version of keys := STIR3.0
+!GENERAL DATA :=
+!GENERAL IMAGE DATA :=
+!type of data := PET
+imagedata byte order := LITTLEENDIAN
+!PET STUDY (General) :=
+!PET data type := Emission
+applied corrections := {None}
+!number format := float
+!number of bytes per pixel := 4
+number of dimensions := 4
+matrix axis label [4] := segment
+!matrix size [4] := 1
+matrix axis label [3] := view
+!matrix size [3] := 32
+matrix axis label [2] := axial coordinate
+!matrix size [2] := { 8}
+matrix axis label [1] := tangential coordinate
+!matrix size [1] := 35
+minimum ring difference per segment := { 0}
+maximum ring difference per segment := { 0}
+Scanner parameters:= 
+Scanner type := unknown
+Number of rings                          := 8
+Number of detectors per ring             := 64
+Inner ring diameter (cm)                 := 88.62
+Average depth of interaction (cm)        := 0.84
+Distance between rings (cm)              := 1.962
+Default bin size (cm)                    := -1
+View offset (degrees)                    := 0
+Maximum number of non-arc-corrected bins := 0
+Default number of arc-corrected bins     := 0
+Energy resolution := 0.22
+Reference energy (in keV) := 511
+Number of blocks per bucket in transaxial direction         := 0
```

Comment:
let's remove all the block stuff. the values are crazy (and not used)

Reply:
same